Makefile

```
@@ -289,7 +287,27 @@ start-jupyter-notebook: build-release-image-jupyter ## Start Jupyter Notebook en
 
 .PHONY: start-jupyter-notebook
 
-debug-jupyter-notebook: build-release-image-jupyter ## Debug jupyter notebook using the ostk-astro package built from current source code, must have run make start-development and ostk-build and ostk-install-python to use this
+debug-jupyter: build-development-image ## Debug jupyter notebook using the ostk-astro package built from current source code
+
+	@ echo "Building Python$(jupyter_python_version) packages..."
+
+	docker run \
+		-it \
+		--rm \
+		--volume="$(CURDIR):/app:delegated" \
+		--workdir=/app/build \
+		$(docker_development_image_repository):$(docker_image_version) \
+		/bin/bash -c "cmake -DBUILD_UNIT_TESTS=OFF -DBUILD_BENCHMARK=OFF -DBUILD_PYTHON_BINDINGS=ON -DPYTHON_SEARCH_VERSIONS="$(jupyter_python_version)" .. \
+		&& $(MAKE) -j $(shell nproc) \
+		&& mkdir -p /app/packages/python \
+		&& rm -rf /app/packages/python/* \
+		&& cp /app/build/bindings/python/dist/*$(extract_python_package_version)*.whl /app/packages/python"
+
+	@ $(MAKE) debug-jupyter-standalone
+
+.PHONY: debug-jupyter
+
+debug-jupyter-standalone: build-release-image-jupyter ## Debug jupyter notebook using the ostk-astro package built from current source code
```

```suggestion
debug-jupyter-rebuild: build-release-image-jupyter ## Debug jupyter notebook using the ostk-astro package built from current source code
```
I would reserve the `standalone` naming for that which is to be used specifically by the CI.